pkg/services/alerting/extractor.go

```diff
@@ -47,15 +50,15 @@ func (e *DashAlertExtractor) lookupQueryDataSource(ctx context.Context, panel *s
 	}
 
 	if dsName == "" && dsUid == "" {
-		query := &models.GetDefaultDataSourceQuery{OrgId: e.OrgID}
-		if err := bus.Dispatch(ctx, query); err != nil {
+		query := &models.GetDefaultDataSourceQuery{OrgId: orgID}
+		if err := e.sqlStore.GetDefaultDataSource(ctx, query); err != nil {
```

Comment:
If I'm not wrong, looks like the `GetDefaultDataSourceQuery` is currently handled by data sources service (`datasource_service.go`), and then it's this one the one that directly calls the `sqlStore` method.

So, it's probably better to do the same here to keep the same behavior.

PS: Looks the that service currently only acts as a simple proxy, but at some point we may add some logic into that "service layer", so would be good to have every caller pointing to the correct method.

Reply:
Yeah, it's a tricky one, especially because we don't have a uniform architecture over all all areas / services.